.changeset/seven-ducks-look.md

```diff
@@ -0,0 +1,24 @@
+---
+'xstate': patch
+---
+
+pr: #3545
+author: @with-heart
```

Comment:
just a note: those are only needed if the desired information doesn't match what can be automatically inferred. For example, I use those when I add changesets after landing a relevant PR.

Reply:
@Andarist is that a `@changesets/changelog-github` feature?

Reply:
> @Andarist is that a @changesets/changelog-github feature?

Yeah, it queries GitHub through the API to infer this info on its own. You only need to supply this manually if you want to override what would be inferred (for whatever reason)